client/.prettierrc.json

```
@@ -1,7 +1,6 @@
-
 {
   "$schema": "https://json.schemastore.org/prettierrc",
-  "semi": false,
+  "printWidth": 100,
   "singleQuote": true,
-  "printWidth": 100
+  "trailingComma": "es5"
```

```suggestion
  "trailingComma": "all"
```
This is also the default and recommended setting, let me know if you think otherwise.